ScintSuite/SimulationCodes/SINPA/_Forw_Mod_xarray.py

```diff
@@ -246,6 +245,7 @@ def synthetic_signal_pr(distro, WF = None, gyrophases = np.pi, plot=False,
         inside the pinhole
     :param  plot: flag to plot the synthetic signals, and the histograms in
         pitch and gyroradius.
+    :param  smooth: smooth the signal of the pinhole. Just for plotting.
```

Comment:
if the smooth only affects the plotting and not the real data, create another routine just for plotting, it is misleading to see one thing and have as output another